cmd/sops/subcommand/filestatus/filestatus.go

```diff
@@ -0,0 +1,47 @@
+package filestatus
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"go.mozilla.org/sops"
+	"go.mozilla.org/sops/cmd/sops/codes"
+	"go.mozilla.org/sops/cmd/sops/common"
+)
+
+// Opts rapresent input option for FileStatus
+type Opts struct {
+	InputStore sops.Store
+	// OutputStore       sops.Store
+	InputPath string
+	// KeyServices       []keyservice.KeyServiceClient
+	// KeyGroups         []sops.KeyGroup
+	// GroupThreshold    int
+}
+
+// Status rapresent file status
+type Status struct {
+	Encrypted bool
+}
+
+func FileStatus(opts Opts) (Status, error) {
+	fileBytes, err := ioutil.ReadFile(opts.InputPath)
+	if err != nil {
+		return Status{}, common.NewExitError(fmt.Sprintf("Error reading file: %s", err), codes.CouldNotReadInputFile)
+	}
+	branches, err := opts.InputStore.LoadPlainFile(fileBytes)
+	if err != nil {
+		return Status{}, common.NewExitError(fmt.Sprintf("Error unmarshalling file: %s", err), codes.CouldNotReadInputFile)
+	}
+	hasMetadata := checkMetadata((branches[0]))
+	return Status{Encrypted: hasMetadata}, nil
+}
+
+func checkMetadata(branch sops.TreeBranch) bool {
```

Comment:
As `ensureNoMetadata` is declared private in `main` package (`cmd/sops/encrypt.go:48`) I duplicated the code, but `checkMetadata` could be in `common` package and being used in `ensureNoMetadata` too to avoid duplication.

Reply:
Eh, this is simple enough that I think it's okay.